templates/etc/apt/sources.list.Debian.j2

```diff
@@ -1,28 +1,43 @@
 # {{ ansible_managed }}
 
+{% if ansible_distribution_major_version  is version('11', '>=') %}
+{% set security_suffix = '-security' %}
+{% else %}
+{% set security_suffix = '/update' %}
+{% endif %}
+
 deb {{ apt_debian_mirror }} {{ ansible_distribution_release }} main
 {{ '# ' if not apt_src_enable else '' }}deb-src {{ apt_debian_mirror }} {{ ansible_distribution_release }} main
 
-deb {{ apt_debian_security_mirror }} {{ ansible_distribution_release }}/updates main
-{{ '# ' if not apt_src_enable else '' }}deb-src {{ apt_debian_security_mirror }} {{ ansible_distribution_release }}/updates main
+deb {{ apt_debian_security_mirror }}/debian-security {{ ansible_distribution_release }}{{security_suffix}} main
```

Comment:
Is `/debian-security` also working for Debian < 11, because this is an addition to the `url`